cache/src/test/java/com/example/expenselogger/cache/repository/ExpenseRepositoryTest.kt

```diff
@@ -41,38 +41,28 @@ internal class ExpenseRepositoryTest {
             val expenseEntity = DummyData.expenseEntity
             val id = expenseRepository.insertExpense(expenseEntity)
             val actual = expenseRepository.getExpense(id)
-            assertThat(actual).isEqualTo(expenseEntity)
+            assertThat(actual).isEqualTo(expenseEntity.copy(id = id))
         }
 
     @Test
     fun `verify that getExpenses gets list of expenses`(): Unit = runBlocking {
         val expense = DummyData.expenseEntity
-        expenseRepository.insertExpense(expense)
+        val id = expenseRepository.insertExpense(expense)
+        val id2 = expenseRepository.insertExpense(expense)
```

Comment:
I don't really understand inserting the expense into the database twice. Is it like to sort of simulate multiple entries?

Reply:
Yes

Reply:
Okay, all clear now. Thank you